Dockerfile

```
@@ -17,6 +17,16 @@ LABEL org.opencontainers.image.source https://github.com/helmfile/helmfile
 
 RUN apk add --no-cache ca-certificates git bash curl jq openssh-client
 
+# Set Helm home variables so that also non-root users can use plugins etc.
+ARG HOME="/root"
```

```suggestion
ARG HOME="/helm"
```

or `/opt/helm`

`/root` is confusing IMO